nf_core/modules/lint/main_nf.py

```diff
@@ -212,22 +212,18 @@ def check_process_section(self, lines):
 
     for l in lines:
         l = l.strip()
-        l = l.replace('"', "")
-        l = l.replace("'", "")
+        l = l.lstrip('"')
+        l = l.lstrip("'")
         if re.search("bioconda::", l):
             bioconda_packages = [b for b in l.split() if "bioconda::" in b]
         if l.startswith("https://containers") or l.startswith("https://depot"):
-            lspl = l.lstrip("https://").split(":")
-            if len(lspl) == 2:
-                # e.g. 'https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img' :
-                singularity_tag = "_".join(lspl[0].split("/")[-1].strip().rstrip(".img").split("_")[1:])
-            else:
-                # e.g. 'https://depot.galaxyproject.org/singularity/fastqc:0.11.9--0' :
-                singularity_tag = lspl[-2].strip()
+            # e.g. 'https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img' :
+            # e.g. 'https://depot.galaxyproject.org/singularity/fastqc:0.11.9--0' :
```

Comment:
Would be nice to also comment what the regex is pulling out of these example strings 👍🏻